corehq/apps/reports/standard/deployments.py

```diff
@@ -663,6 +668,55 @@ class AggregateUserStatusReport(ProjectReport, ProjectReportParametersMixin):
     def decorator_dispatcher(self, request, *args, **kwargs):
         super(AggregateUserStatusReport, self).decorator_dispatcher(request, *args, **kwargs)
 
+    @property
+    @memoized
+    def selected_app_id(self):
+        return self.request_params.get(SelectApplicationFilter.slug, None)
+
+    def _get_histogram_aggregation_for_app(self, nested_field_name, date_field_name):
```

Comment:
nit: I believe the parameter should be called "field_name" only. Its not clear to me why its called "nested_" already.

Reply:
I called this `nested_field_name` to try and make it easy to identify that this is a nested field in the `users` ES index. Happy to change this if it is causing more confusion than it's helping.

Reply:
[9aaabbf](https://github.com/dimagi/commcare-hq/pull/34178/commits/9aaabbfb5ae31c4d87cc4f20fc293b69026b7043)